src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -425,6 +428,11 @@ private bool CanContinueBatch(ReplicationState state, ref long next)
                 AssertNotClusterTransactionDocumentForLegacyReplication(state.Item);
             }
 
+            if (_parent.SupportedFeatures.Replication.IncrementalTimeSeries == false)
```

Comment:
We enter here only if `replicationState.LastTransactionMarker != item.TransactionMarker`, when I think about it all of those assertion are in the wrong place?

What if I change a doc and increment a TS in the same session, I will not fail here, right?

Reply:
you are correct
added a failing test : https://github.com/ravendb/ravendb/pull/13040/commits/db788a7a35d85ea055dd7ff0dcb942c1ddecf0d7

Reply:
We need to fix this and backport the fix... all of those assertions will not be triggered under the above conditions

Reply:
handled in a separate PR : https://github.com/ravendb/ravendb/pull/13056